backend/base/templates/slot-list.html

```diff
@@ -224,6 +197,12 @@ <h6 class="mb-5">Available Slots are:</h6>
                 });
             }
         });
+        const $successAlert = $('#successAlert');
+        if (successAlert.length) {
+            setTimeout(() => {
+                successAlert.hide();
+            }, 2500);
+        }
     </script>

```

Comment:
check this function whether its working. constant is defined with $ symbol, but used without symbol inside function.